### PR TITLE
Translate documents in simplified Chinese

### PR DESCRIPTION
**Pull request by addozhang**
Referring to [comment](https://github.com/openservicemesh/osm-docs/pull/384#issuecomment-1194643775) in #384 , create a new PR for translate v1.2 docs in simplified Chinese.

### CODE REVIEW

**Comment by zr-msft** [comment]
thank you @addozhang 

Did you copy the content from #384? or did you redo the translation based on the latest content in release-v1.2?

**Comment by addozhang** [comment]
I compared release-v1.2 with release-v1.1 and did the translation of new commits. 

Now the PR is for translation based on release-v1.2.

File: content/zh/docs/getting_started/traffic_split.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
+---
+title: "配置流量拆分"
+description: "使用 SMI Traffic Split API 来平衡服务之间的流量"
+type: docs
+weight: 4
+---
+
+# 配置两个服务之间的流量拆分
+
+我们将演示如何平衡两个 Kubernetes 服务之间的流量，通常被称为流量拆分。我们将在后端 `bookstore` 服务和 `bookstore-v2` 服务之间来拆分原来定向到*根* `bookstore` 服务的流量。`bookstore` 和 `bookstore-v2` 服务被称为椰子服务。有关如何配置服务的流量拆分，请参阅 [流量拆分的 how-to 指南](/docs/guides/traffic_management/traffic_split.md)。
```
Comment (allenlsy) [inline content/zh/docs/getting_started/traffic_split.md:10]: 椰子服务

File: content/zh/docs/getting_started/traffic_split.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+部署 SMI 流量拆分策略来定向 100% 发送到根 `bookstore` 服务的流量到 `bookstore-v1` 服务后端。这将确保发送到 `bookstore` service 的流量只会发送到 `version v1` 的 bookstore 应用，即 `bookstore-v1` service 的 pod。随后更新 `TrafficSplit` 配置来发送部分流量到使用了 `bookstore-v2` 叶子服务的 `version v2` bookstore 服务。
+
+因此，如果需要流量拆分，确保客户端应用程序始终与根服务通信非常重要。否则，当需要流量拆分时，需要更新客户端应用程序以与 *根* 服务通信。
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/openservicemesh/osm-docs/{{< param osm_branch >}}/manifests/split/traffic-split-v1.yaml
+```
+
+_注意：根服务是个 Kubernetes 服务，其标签选择器需要与叶子服务相匹配。在本示例中，根服务 `bookstore` 使用 `app:bookstore` 选择器分别匹配 `bookstore (v1)` 和 `bookstore-v2` deployment 的标签 `app:bookstore,version:v1` 和 `app:bookstore,version=v2`。在 SMI 流量拆分里，根服务能够作为有或者没有 `.<namespace>` 后缀的服务名称被引用。_
```
Comment (allenlsy) [inline content/zh/docs/getting_started/traffic_split.md:48]: `version=v2` => `version:v2`

File: content/zh/docs/api_reference/config/v1alpha2.md [hunk 3]
```diff
@@ -0,0 +1,1416 @@
+---
+title: "配置 v1alpha2 API 参考"
+description: "配置 v1alpha2 API 参考文档."
+type: docs
+---
+
```
Comment (allenlsy) [inline content/zh/docs/api_reference/config/v1alpha2.md:6]: FYI: not translated.

File: content/zh/docs/getting_started/traffic_split.md (hunk 2 above)
Comment (addozhang) [reply content/zh/docs/getting_started/traffic_split.md]: Above two were fixed.

**Review by allenlsy — APPROVED** [review]

**Comment by addozhang** [comment]
@zr-msft Hi dude, how is the review going?

**Comment by zr-msft** [comment]
@addozhang Thank you for following up. Did you incorporate the feedback from @allenlsy ?

**Comment by zr-msft** [comment]
LGTM @addozhang

**Review by zr-msft — APPROVED** [review]

**Comment by addozhang** [comment]
Thanks @zr-msft. So now we need have one more approver to approve this PR, rt?

**Review by keithmattix — APPROVED** [review]

**Comment by keithmattix** [comment]
Approved! Since @zr-msft already approved, I'll go ahead and merge this PR 👍🏾 Thanks for this awesome work @addozhang!

**Comment by addozhang** [comment]
Thanks @keithmattix. I will follow changes in 1.2.1 and future release too.

**Pull request merged**